src/main/java/com/example/rets_api/service/PropertyService.java

```diff
@@ -41,4 +41,17 @@ public List<PropertyDTO> getPropertiesByParams(PropertyFilter propertyFilter){
         return propertiesDTO;
     }
 
+    public List<PropertyDTO> getAllProperties() {
```

Comment:
Don't add this back, we don't need anymore

Reply:
oh ok sure

Reply:
it was auto merging I think, sorry